lib/rubocop/cop/rails/content_tag.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that `tag` is used instead of `content_tag`
+      # because `content_tag` is legacy syntax.
+      #
+      # @example
+      #  # bad
+      #  content_tag(:p, 'Hello world!')
+      #  content_tag(:br)
+      #
+      #  # good
+      #  tag.p('Hello world!')
+      #  tag.br
+      class ContentTag < Cop
+        include RangeHelp
+        extend TargetRailsVersion
+
+        minimum_target_rails_version 5.1
+
+        MSG = 'Use `tag` instead of `content_tag`.'
+
+        def on_send(node)
+          return unless node.method?(:content_tag)
+
+          add_offense(node)
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            if method_name?(node.arguments.first)
```

Comment:
Can you use `node.first_argument`?

```suggestion
            if method_name?(node.first_argument)
```